excel/src/main/java/org/apache/metamodel/excel/ExcelUpdateCallback.java

```diff
@@ -78,7 +87,24 @@ public ExcelDataContext getDataContext() {
 
     protected void close() {
         if (_workbook != null) {
-            ExcelUtils.writeWorkbook(_dataContext, _workbook);
+
+            final Resource resource;
+            if(_dataContext.getResource() instanceof FileResource){
```

Comment:
Trying to find what the issue might be ... And this if-sentence is making me wonder: Why is it here, and if it is supposed to write non-file resources to a temp-file, isn't it all wrong then? Shouldn't it be "if resources is NOT a FileResource"?